plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -1483,6 +1483,24 @@ public void testArraySchemas()
                 "  }" +
                 "} ");
         tester.testRoundTrip(getStandardListObjectInspector(javaIntObjectInspector), values, values, "my_list", new ArrayType(INTEGER), Optional.of(customNamingSchema));
+
+        MessageType optimizedParquetWriterOldListSchema = parseMessageType("message trino_schema {" +
+                "  optional group my_list (LIST){" +
+                "    repeated group list {" +
+                "        optional int32 array;" +
+                "    }" +
+                "  }" +
+                "} ");
+        tester.testRoundTrip(getStandardListObjectInspector(javaIntObjectInspector), values, values, "my_list", new ArrayType(INTEGER), Optional.of(optimizedParquetWriterOldListSchema));
+
+        MessageType optimizedParquetWriterListSchema = parseMessageType("message trino_schema {" +
```

Comment:
This seems to be a repeat of `sparkSchema` a few lines above. So we can skip this and note in a comment there that this is the current schema used by the Trino optimized parquet writer